reconcile/utils/output.py

```diff
@@ -26,6 +26,8 @@ def print_output(
         print(json.dumps(content))
     elif output == "yaml":
         print(yaml.dump(content))
+    elif output == "return":
+        return content
```

Comment:
I don't think a function called `print_output` should either print or return stuff based on arguments... you may want to create a separate function that returns content used by this one...